msg/en_US/utils.msg

```diff
@@ -978,9 +978,9 @@ usage: %1$s applyinfo [OPTION] database-name\n\
 \n\
 valid options:\n\
   -r, --remote-host-name      remote host name; display remote node's active log information \n\
-  -a, --applied-info 	      display applied information \n\
+  -a, --applied-info 	      display applied information; the -L option is required \n\
   -L, --copied-log-path=PATH  path of copied log volumes; display copied log information \n\
```

Comment:
```suggestion
  -L, --copied-log-path=PATH  path of copied log volumes; display copied log information \n\
  -a, --applied-info 	      display applied information; the -L option is required \n\
```

how about change the sequence ? 
It seems good to first describe the L option, and then describe the a and p options that depend on the L option.

Reply:
I think so. It will be more readable. Thank you